required/amsmath/amsmath-2018-12-01.sty

```diff
@@ -27,7 +27,7 @@
 %%
 \NeedsTeXFormat{LaTeX2e}% LaTeX 2.09 can't be used (nor non-LaTeX)
 [1994/12/01]% LaTeX date must be December 1994 or later
-\ProvidesPackage{amsmath}[2018/12/01 v2.17b AMS math features]
+\ProvidesPackage{amsmath}[2018/12/01 v2.17b AMS math features (updated for rollback)]
```

Comment:
I'd have preferred a version change: would help in the tests!

Reply:
In this particular case on could change the version (but not the date) because amsmath goes
```
\DeclareRelease{}{1994-06-01}{amsmath-2018-12-01.sty}
\DeclareRelease{}{2018-12-01}{amsmath-2018-12-01.sty}
\DeclareCurrentRelease{}{2019-04-01}
```
i.e., it does not allow you to roll back to a version but only to a date. However, I don't think it is making a good example if we do. In other packages there will be both a version string as well as a date and so both are really part of the rollback syntax and you would affect rollback in documents if you change either of them. (same reason why can't bump the \CurrentVersion date up).

Reply:
Awkward but I see the point - otherwise we good to go here?

Reply:
> I'd have preferred a version change: would help in the tests!

it would help us perhaps but I think it's confusing to roll back to a version that didn't previously exist (even if that's what this is, really)

Reply:
I don't see that we are rolling back to a different version or date here. We are rolling back to the version current on that date (only we have corrected how this is happening. If You install a TL from that data/year it would say 2.17b and all we did adjust the rollback so the version as it was then comes back to life.

so in my opinion it is as it should be and the comment in the \ProvidesPackage is really the correct way to handle it.